mlir/lib/Dialect/Rock/Transforms/GemmToGridwise.cpp

```diff
@@ -139,7 +139,7 @@ static LogicalResult commonAttentionGemmElmtGemm(
   if (!isa<MemRefType>(op.getAType()))
     return op.emitOpError("Cannot lower unbufferized gemm to gridwise");
 
-  bool isAccel = rock::isAccel(op.getGemmFeatures());
+  bool isAccel = rock::isAccel(rock::getFeatures(op));
```

Comment:
It would need to compute features multiple times now v/s earlier it was just available was an attribute to look up directly. Can possibly slow down compilation perhaps. we can try to cache it somehow

Reply:
Better thing would what daniel is suggesting is that we compute  once and put that on func as an attribute. 

We can keep that as a follow up. that is fine ..